lib/graphql/tracing/trace.rb

```diff
@@ -15,6 +15,7 @@ def initialize(multiplex: nil, query: nil, **_options)
         @query = query
       end
 
+      # The Ruby parser doesn't call this method (`graphql/c_parser` does.)
```

Comment:
❤️ the new parser

Wish this behaviour was more widely communicated given the Ruby parser is the default (our CI started failing)

Reply:
Hey, sorry for the nasty surprise -- at least I added a changelog entry for this change in 7c3d899f0